README.md

```diff
@@ -101,6 +101,18 @@ Run tests:
 ```bash
 docker-compose exec app pytest --cov=src --cov=migrations/versions tests
 ```
+### Aggregate MDS
+For local development on OSX, install elasticsearch and ensure it is running.
+
+testing populate:
+```bash
+python src/mds/populate.py --config <config file> --hostname localhost --port 9200
```

Comment:
Need to update documentation since the command line arguments are no longer used